lib/logger.js

```diff
@@ -0,0 +1,26 @@
+// Dummy logger
+// Should be replaced with a real logger if fh-db is beeing used in the
+// future.
+
+var logCfg = {
+  error: true,
+  warn: false,
+  warning: false,
+  info: false,
+  debug: false
+};
+
+var doLog = function(level, prefix, msg) {
+  if( logCfg[level] ) {
+    // eslint-disable-next-line no-use-before-define
+    console.log(prefix, msg);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)